azerothcore/azerothcore-wotlk
# fix(DB/skilllineability_dbc): Correct structure of skilllineability_dbc

## Changes Proposed:
-  Aligns structure of DB table more to possible DBC structure.

## Issues Addressed:

- Finishes #4312

## SOURCE:

#7513

## Tests Performed:

- Ran with contents inside the table without crashing.

## How to Test the Changes:


1. Run the query.
2. Run this query:
```INSERT INTO `skilllineability_dbc` (`ID`, `SkillLine`, `Spell`, `RaceMask`, `ClassMask`, `MinSkillLineRank`, `SupercededBySpell`, `AcquireMethod`, `TrivialSkillLineRankHigh`, `TrivialSkillLineRankLow`, `CharacterPoints_1`, `CharacterPoints_2`, `TradeSkillCategoryID`) VALUES (69, 6, 116, 0, 128, 0, 0, 1, 0, 0, 0, 0, 0);```
3. Observe no crash.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
